docs/reST/tut/MoveIt.rst

```diff
@@ -449,6 +452,127 @@ here it may not matter, but when objects are overlapping, using two loops
 like this becomes important.
 
 
+Preparing for Improved User Input
+---------------------------------
+
+With all keyboard input terminating the program, that's not very interactive.
+Let's add some extra user input!
+
+First we should create a unique character that the player will control. We
+can do that in much the same way we created the other movable entities. Let's
+call the player object p. We can already move any object, but, a player should
+have more input than simply moving right. To accommodate this, let's revamp
+our move function under our GameObject class. ::
+
+  >>> def move(self, up=False, down=False, left=False, right=False):
+  ...   if right:
+  ...       self.pos.right += self.speed
+  ...   if left:
+  ...       self.pos.right -= self.speed
+  ...   if down:
+  ...       self.pos.top += self.speed
+  ...   if up:
+  ...       self.pos.top -= self.speed   
+  ...   if self.pos.right > 640:
+  ...       self.pos.left = 0
+  ...   if self.pos.top > 420:
+  ...       self.pos.top = 0
+  ...   if self.pos.right < 79:
+  ...       self.pos.right = 640
+  ...   if self.pos.top < 0:
+  ...       self.pos.top = 420
```

Comment:
Why are there hardcoded magic integers here? It would be better to use constants, since this is a newbie guide we need to introduce readers to some good programming practices here ;)